<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recorderClass</class>
 <widget class="QMainWindow" name="recorderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>356</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>342</width>
    <height>356</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>342</width>
    <height>356</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>recorder</string>
  </property>
  <widget class="QWidget" name="panel">
   <widget class="VideoGLWidget" name="videoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Tracker</string>
    </property>
    <widget class="QPushButton" name="saveTrackerConfig">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save configuration</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showTracker">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Configure...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Recorder</string>
    </property>
    <widget class="QLineEdit" name="aviFilename">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>tracker_%s.avi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recordAVI">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Record AVI</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>AVI Filename</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoGLWidget</class>
   <extends>QWidget</extends>
   <header>videoglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="recorder.qrc"/>
 </resources>
 <connections/>
</ui>
